packages/json/test/Json.test.ts

```diff
@@ -121,6 +121,6 @@ describe('Json', () =>
 
         const data = readFileSync(`${outputDir}/json/json.json`, 'utf8');
 
-        expect(data.replace(/\\/g, '').trim()).toEqual(`"{"hello":"world","Im":"not broken"}"`);
+        expect(data.replace(/\\/g, '').trim()).toEqual(`{\"hello\":\"world\",\"Im\":\"not broken\"}`);
```

Comment:
is this correct?
There was an issue where the json called `stringify` twice, resulting in the file having these extra escape chars

Reply:
oh, yes - let me tweak. no need for the \ anymore!